index.php

```diff
@@ -25,20 +25,36 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
 	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $lang ?>" lang="<?php echo $lang ?>">
+<!-- TODO FIND PERMANENT PROPER SOLUTION FOR THIS. DO NOT MERGE / DEPLOY. -->
+<div id="WMDE-Banner-Container"
+     data-trackingparams="piwik_campaign=wpde-02-181120&amp;piwik_kwd=wpde-02-181120-top-ctrl"
+     data-campaign-tracking="wpde-02-181120"
+     data-tracking="wpde-02-181120-top-ctrl"
+></div>
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 	<title>wikipedia.de - Wikipedia, die freie Enzyklop&auml;die</title>
 	<link rel="apple-touch-icon" href="/img/wikipedia.png" />
 	<link rel="stylesheet" media="screen" type="text/css" href="style.css" />
-	<script type="text/javascript" src="js/jquery-1.11.3.min.js"></script>
-	<?php
-		if ( isset($banner) && $banner ):
-	?>
-		<script type="text/javascript" src="js/jquery.cookie.min.js"></script>
-	<?php
-		endif;
-	?>
+    <script type="application/javascript">
+        // TODO FIND PERMANENT PROPER SOLUTION FOR THIS. DO NOT MERGE / DEPLOY.
+		GlobalBannerSettings = {
```

Comment:
The GlobalBanner settings still have to be present on the page, this should also somehow be incorporated into the banner server I guess, let's talk about that.